JSONSchema/genesis-filler-schema.json

```diff
@@ -0,0 +1,129 @@
+{
+    "definitions": {
+        "MiningMethod": {
+            "anyOf": [
+                {
+                    "pattern": "NoProof",
+                    "type": "string"
+                }
+            ]
+        },
+        "PreStateAccount": {
```

Comment:
if its a preState account in tests then all fields are required. 
if its a prestate account in Genesis then  it has different scheme:
```
    if (_obj.count(c_precompiled))
    {
        // A precompiled contract
        requireJsonFields(_obj, "validateAccountObj",
            {{c_precompiled, {{js::obj_type}, JsonFieldPresence::Required}},
                {c_wei, {{js::str_type}, JsonFieldPresence::Optional}}});    
    }
    else if (_obj.size() == 1)
    {
        // A genesis account with only balance set
        if (_obj.count(c_balance))
            requireJsonFields(_obj, "validateAccountObj",
                {{c_balance, {{js::str_type}, JsonFieldPresence::Required}}});
        else
            requireJsonFields(_obj, "validateAccountObj",
                {{c_wei, {{js::str_type}, JsonFieldPresence::Required}}});
    }
```

so it could have only balance defined.

Reply:
@winsvega not sure what you're suggesting here. Currently none of the fields in `PreStateAccount` are required, are you saying we should require `balance`?

Reply:
hm. thats a good question. I think we should require balance. 
account nonce is default. account code empty by default. same with storage. 
but if we not declare any fields then what is the meaning of putting account into genesis?